diff --git a/arch/arm64/boot/dts/vendor/qcom/kona-v2.dtsi b/arch/arm64/boot/dts/vendor/qcom/kona-v2.dtsi
index 05920c691faf..e743538a9a1d 100755
--- a/arch/arm64/boot/dts/vendor/qcom/kona-v2.dtsi
+++ b/arch/arm64/boot/dts/vendor/qcom/kona-v2.dtsi
@@ -380,7 +380,6 @@
 	interrupt-names = "dcvsh0_int", "dcvsh1_int", "dcvsh2_int";
 
 	qcom,cpufreq-table-0 =
-			< 300000>,
 			< 691200>,
 			<1171200>,
 			<1708800>,
@@ -388,11 +387,8 @@
 
 	qcom,cpufreq-table-1 =
 			< 825600>,
-			<1056000>,
-			<1382400>,
 			<1478400>,
 			<1766400>,
-			<1958400>,
 			<2054400>,
 			<2246400>,
 			<2342400>,
@@ -400,11 +396,8 @@
 
 	qcom,cpufreq-table-2 =
 			< 844800>,
-			<1401600>,
 			<1862400>,
 			<2073600>,
-			<2265600>,
-			<2361600>,
 			<2457600>,
 			<2553600>,
 			<2649600>,
